pkgs/development/python-modules/gst-python/default.nix

```diff
@@ -42,13 +45,6 @@ buildPythonPackage rec {
     })
   ];
 
-  postPatch = lib.optionalString stdenv.hostPlatform.isDarwin ''
-    # The analytics tests often timeout under load on Darwin (e.g. on Hydra), so remove them
-    substituteInPlace testsuite/meson.build --replace-fail \
-      "['Test analytics', 'test_analytics.py', ['gst-plugins-bad/gst-libs/gst/analytics', 'gst-plugins-base/gst-libs/gst/video']]," \
-      ""
-  '';
-
   # Python 2.x is not supported.
   disabled = !isPy3k;
```

Comment:
I'm not sure we need these two lines anymore. The set of supported python 2.x packages here is tiny.

Reply:
fwiw [~5 years ago](https://github.com/NixOS/nixpkgs/pull/99345/files#diff-20b4fe29f32546130ecfde66c9b8eee1b686f41edf317c8781625a77561f2ac7R27-R29) and [~64 total instances left](https://github.com/search?q=repo%3ANixOS%2Fnixpkgs%20%22disabled%20%3D%20!isPy3k%22&type=code)